### ISSUE TITLE
SimpleServerAndClientTest is flaky

### ISSUE THREAD

**Issue by nicktindall** [OpenHFT/Chronicle-Network#133]
It fails semi-regularly.

I suspect the root cause is a NullPointerException that seems to get logged (at fatal log level?), after that whatever failed causes a timeout

```
[14:24:02]F:			 [test] java.lang.NullPointerException
[14:24:02]F:			 [test] java.lang.NullPointerException
[14:24:02]i:	 [Step 1/2] net.openhft.performance.tests.network.VerySimpleClientTest
[14:24:02]i:		 [net.openhft.performance.tests.network.VerySimpleClientTest] test
```

https://teamcity.chronicle.software/test/-8391990066415196831?currentProjectId=OpenHFT_ChronicleNetwork

**Comment by JerryShea** [comment]
Different exception now - only on Java 11 build: `java.lang.AssertionError: Reference counted not released`

**Comment by hft-team-city** [comment]
Released in [Chronicle-Network-2.23ea4](https://github.com/OpenHFT/Chronicle-Network/releases/chronicle-network-2.23ea4), [BOM-2.23ea37](https://github.com/OpenHFT/OpenHFT/releases/chronicle-bom-2.23ea37)
